src/entrypoints/publishing/router/publishing/pages/service-messages-page/index.tsx

```diff
@@ -48,6 +48,12 @@ const ServiceMessagesPage: FC<Props> = () => {
               (!valid_to ||
                 new Date().getTime() <= new Date(valid_to).getTime()))
         )
+        .sort(({ valid_from: fromA }, { valid_from: fromB }) => {
```

Comment:
Foretrekker også her at sorteringen gjøres direkte i spørringen mot Strapi.
Det er ikke lagt inn mulighet for det nå, men ut fra dokumentasjon fra Strapi så skal det være mulig. Dvs. skal kunne defineres i filen /src/services/api/strapi/graphql/service-message.graphql og deretter generere på nytt, så skal det bli tilgjengelig som en parameter i useGetServiceMessagesQuery.